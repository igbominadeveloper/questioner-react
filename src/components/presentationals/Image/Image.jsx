import React from 'react';

const Image = ({ url }) => {
  return <img src={url} style={{ height: '100%', width: '100%' }} />;
};

export default Image;
