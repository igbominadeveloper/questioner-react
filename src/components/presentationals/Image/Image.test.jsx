import React from 'react';
import { shallow } from 'enzyme';

import Image from './Image';

const props = {
  url: '',
};

describe('Image', () => {
  describe('Render', () => {
    it('renders without error', () => {
      const component = shallow(<Image {...props} />);
      expect(component).toMatchSnapshot();
    });
  });
});
