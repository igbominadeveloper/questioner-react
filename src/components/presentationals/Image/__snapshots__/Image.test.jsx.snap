// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Image Render renders without error 1`] = `
<img
  src=""
  style={
    Object {
      "height": "100%",
      "width": "100%",
    }
  }
/>
`;
